api/metrics-api-jaxrs-1.1/src/main/java/org/hawkular/metrics/api/jaxrs/config/ConfigurationKey.java

```diff
@@ -26,6 +26,7 @@
  */
 public enum ConfigurationKey {
 
+    ALLOWED_CORS_ORIGINS("hawkular.metrics.allowed-cors-origins", "*", "ALLOWED_CORS_ORIGINS", false),
```

Comment:
The JIRA request was to constrain the origin checking in restrictive environments; hence, the admin now has an option to configure allowed origins. 

Since the CORS setup really applies to browser requests, I favor allow all by default unless further configured by the admin.